### ISSUE TITLE
Voice Changer chugs a lot when starting up cold, then weirdly gets better over time.

### ISSUE THREAD

**Issue by n0753w** [w-okada/voice-changer#394]
### Issue Type

Others

### vc client version number

MMVCServerSIO_win_onnxgpu-cuda_v.1.5.3.7.zip

### OS

Windows 10 Home

### GPU

GTX 1650 GDDR6 4gb

### Clear setting

no

### Sample model

yes

### Input chunk num

yes

### Wait for a while

The GUI successfully launched.

### read tutorial

yes

### Voice Changer type

RVC

### Model type

pyTorch

### Situation

The voice changer works great when it works. The problem is that whenever I boot up the program cold (haven't opened it in a while), the voice playback is **really** choppy. Changing chunk doesn't work, switching to dio or Crepe doesn't fix it. The only way for it to be fixed is to basically just sit back and hope it works out itself. I find that turning the program off and on again sort of fixes it, but I repeated that action only to have to reboot it multiple times before it actually started working properly again. Is this due to my rather old GPU? Is it because I stored the whole file in a portable SSD instead of my main NVME drive? 

tldr: program works when it feels like working. When it does work, I can go to about 128 chunks. When it doesn't, it just sounds like choppy cursed stuff. How do you make it so that it consistently boots up ready-to-work?

**Comment by w-okada** [comment]
We cannot provide a sure-fire way to work in all user environments. Please try different approaches.

**Comment by n0753w** [comment]
The latest update (.8a) fixed the issue. Thanks.
